README.md

```diff
@@ -1,45 +1,51 @@
+[Publisher analysis template](https://github.com/open-contracting/notebooks-ocds/blob/main/publisher_analysis_template.ipynb) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/open-contracting/notebooks-ocds/blob/main/publisher_analysis_template.ipynb) | Use this notebook to analyse data from a specific publisher 
+[Meta analysis template](https://github.com/open-contracting/notebooks-ocds/blob/main/meta_analysis_template.ipynb) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/open-contracting/notebooks-ocds/blob/main/meta_analysis_template.ipynb) | Use this notebook to analyse data from multiple publishers, or to perform other types of analysis on the Kingfisher database
+[Structure and format feedback template](https://github.com/open-contracting/notebooks-ocds/blob/main/structure_and_format_feedback_template.ipynb) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/open-contracting/notebooks-ocds/blob/main/structure_and_format_feedback_template.ipynb) | Use this notebook to provide feedback on structure and format issues reported by the Data Review Tool
+[Data quality feedback template](https://github.com/open-contracting/notebooks-ocds/blob/main/data_quality_feedback_template.ipynb) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/open-contracting/notebooks-ocds/blob/main/data_quality_feedback_template.ipynb) | Use this notebook to provide detailed feedback on structure and format, conformance and coherence issues 
+Usability checks template | Use this notebook to provide feedback on data usability
 
-To open a component from a different branch, use Colab's [GitHub browser](https://colab.research.google.com/github/open-contracting/kingfisher_notebook_components/) to choose the component and branch you want to open.
+## Contributing
 
-Alternatively, you can use the Open in Colab browser extension ([Chrome](https://chrome.google.com/webstore/detail/open-in-colab/), [Firefox](https://addons.mozilla.org/en-US/firefox/addon/open-in-colab/)) to add a button that, when clicked when viewing a Jupyter notebook on GitHub, will open that notebook in Colab.
+### Components
 
-## Templates
+To ease maintenance, the notebooks are made up of reusable components. The following table describes the relationship between the components and the notebooks.
 
-Each time a component is updated, GitHub Actions builds template notebooks from the components and writes them to a [Google Drive folder](https://drive.google.com/drive/u/0/folders/1eb3pSQ55HylMsmwKqu7MrkvH12ROD4-9).
+Component | Tasks | Used in
```

Comment:
I don't think the "Used in" column adds much that a contributor can't simply read in manage.py, and it adds maintenance burden to keep that column in sync. I suggest removing it.

Reply:
Done in https://github.com/open-contracting/notebooks-ocds/pull/30/commits/c584451718fab92970860c6ce2354ce2e4726972